app/main.py

```diff
@@ -1,6 +1,21 @@
-from typing import Callable
+from typing import Any, Callable
+
+data = []
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    def inner(*args, **kwargs) -> Any:
```

Comment:
Use a decorator from `functools` to preserve the wrapped function name and documentation